docs/manual/src/internals/lifting_and_lowering.md

```diff
@@ -85,6 +86,7 @@ The details of this format are internal only and may change between versions of
 | `sequence<T>` | Serialized `i32` item count followed by serialized items; each item is a serialized `T` |
 | `record<string, T>` | Serialized `i32` item count followed by serialized items; each item is a serialized `string` followed by a serialized `T` |
 | `enum` | Serialized `u32` indicating variant, numbered in declaration order starting from 1 |
+| `[TaggedUnion] interface` | Serialized `u32` indicating variant as per `enum`, followed by the serialized value of each field, in declaration order |
```

Comment:
I don't think "union" and "enum" are simple synonyms in rust - is `TaggedEnum` a better name?

Reply:
Yeah, that's a good point re: the likelihood the name here will be interpreted somewhere close to Rust's semantics.

Reply:
I spent a good deal of time thinking about this yesterday, and still couldn't remember `TaggedUnion`. Having said that: what a lovely bikeshed!

Reply:
I'm going to try just `[Enum]` and see how I like it...